tests/UnifiedSpecTests/UnifiedTestRunner.php

```diff
@@ -354,31 +352,7 @@ private function isClientSideEncryptionSupported(): bool
             return false;
         }
 
-        return static::isCryptSharedLibAvailable() || static::isMongocryptdAvailable();
-    }
-
-    private static function isCryptSharedLibAvailable(): bool
-    {
-        $cryptSharedLibPath = getenv('CRYPT_SHARED_LIB_PATH');
-
-        if ($cryptSharedLibPath === false) {
-            return false;
-        }
-
-        return is_readable($cryptSharedLibPath);
-    }
-
-    private static function isMongocryptdAvailable(): bool
-    {
-        $paths = explode(PATH_SEPARATOR, getenv("PATH"));
-
-        foreach ($paths as $path) {
-            if (is_executable($path . DIRECTORY_SEPARATOR . 'mongocryptd')) {
-                return true;
-            }
-        }
-
-        return false;
+        return FunctionalTestCase::isCryptSharedLibAvailable() || FunctionalTestCase::isMongocryptdAvailable();
```

Comment:
This is one thing I had in mind when creating [PHPLIB-1170](https://jira.mongodb.org/browse/PHPLIB-1170).